patterns/molecules/postcard/postcard.html.twig

```diff
@@ -7,7 +7,7 @@
 {% set attributes = attributes.addClass(['postcard', variants.image_orientation, variants._text_orientation]) %}
 
 <div{{ attributes }}> {# has-image-right, has-text-offset, has-text-inset #}
-  <div class="postcard__image" style="background-image:url('{{ image|get_img_src }}')"></div>
+  <div class="postcard__image b-lazy bg" data-src="{{ image|get_img_src }}"></div>
```

Comment:
@aaroncole same oddity as the footer. I see the classes you have added, but still see it as 'style='background-image...'